templates/404.html

```diff
@@ -0,0 +1,164 @@
+                          <path vector-effect="non-scaling-stroke" d="M77,88.5 L92.5,88.5 L92.5,81 C92.5,77.1340068 95.6340068,74 99.5,74 L99.5,74 C103.365993,74 106.5,77.1340068 106.5,81 L106.5,96.5 C106.5,99.8137085 103.813708,102.5 100.5,102.5 L77,102.5"
+                            class="slds-illustration__fill-secondary" stroke-linecap="round" stroke-linejoin="round"></path>
+                        </g>
+                      </g>
+                      <g transform="translate(429.000000, 242.000000)" class="slds-illustration__stroke-secondary" stroke-linecap="round" stroke-width="3">
+                        <g>
+                          <path vector-effect="non-scaling-stroke" d="M12.5,4 L12.5,50"></path>
+                          <polyline vector-effect="non-scaling-stroke" stroke-linejoin="round" points="11 18.5 0.5 18.5 0.5 0"></polyline>
+                          <polyline vector-effect="non-scaling-stroke" stroke-linejoin="round" points="14 25.5 24.5 25.5 24.5 8"></polyline>
+                          <polyline vector-effect="non-scaling-stroke" stroke-linejoin="round" points="2.5 31 2.5 38.5 11 38.5"></polyline>
+                        </g>
+                      </g>
+                      <g transform="translate(327.000000, 95.000000)">
+                        <g>
+                          <circle vector-effect="non-scaling-stroke" class="slds-illustration__fill-secondary" cx="64" cy="64" r="23"></circle>
+                          <circle vector-effect="non-scaling-stroke" class="slds-illustration__stroke-secondary" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" stroke-dasharray="135,1,1,18" transform="translate(64.000000, 64.000000) rotate(230.000000) translate(-64.000000, -64.000000) "
+                            cx="64" cy="64" r="45"></circle>
+                          <circle vector-effect="non-scaling-stroke" class="slds-illustration__stroke-secondary" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" stroke-dasharray="107,10" transform="translate(64.000000, 64.000000) rotate(150.000000) translate(-64.000000, -64.000000) "
+                            cx="64" cy="64" r="33"></circle>
+                        </g>
+                        <g transform="translate(41.000000, 41.000000)" class="slds-illustration__stroke-primary" stroke-width="3">
+                          <circle vector-effect="non-scaling-stroke" cx="23" cy="23" r="23"></circle>
+                        </g>
+                      </g>
+                    </g>
+                  </g>
+                </g>
+              </svg>
+            </div>
+            <h3 class="slds-illustration__header slds-text-heading_medium">
+              ¯\_(ツ)_/¯
+            </h3>
+            <p class="slds-text-body_regular">That page cannot be found. Try the <a href="/">home page</a>?</p>
```

Comment:
@jerivas I wonder if these user-facing strings should go through Django translation?

Reply:
I'll generate the new strings now 🔠